http/spec_conformance_test.go

```diff
@@ -45,6 +46,7 @@ func TestSpecConformance(t *testing.T) {
 		"DOMAIN_RANDAO":                         phase0.DomainType{},
 		"DOMAIN_SELECTION_PROOF":                phase0.DomainType{},
 		"DOMAIN_VOLUNTARY_EXIT":                 phase0.DomainType{},
+		"DOMAIN_APPLICATION_BUILDER":            phase0.DomainType{},
```

Comment:
Not sure if you'd like a new folder called spec/v1 with file types.go in it for this and change phase0.DomainType{} to V1.DomainType{}

domain type reference can be found below

https://github.com/ethereum/builder-specs/blob/main/specs/builder.md#validatorregistrationv1

Reply:
I also noticed this 

"Note: DOMAIN_APPLICATION_MASK reserves the rest of the bitspace in DomainType for application usage. This means for some DomainType DOMAIN_SOME_APPLICATION, DOMAIN_SOME_APPLICATION & DOMAIN_APPLICATION_MASK MUST be non-zero. This expression for any other DomainType in the consensus specs MUST be zero."

on https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md

which sounds like both DOMAIN_APPLICATION_BUILDER & DOMAIN_APPLICATION_MASK need to be set 

not sure if this has any impact on required code changes